internal/verifier/attestation.go

```diff
@@ -136,6 +136,7 @@ func CreateAttestations(ctx context.Context, input rebuild.Input, finalStrategy
 			return nil, nil, errors.Wrap(err, "marshalling manual strategy")
 		}
 		externalParams["manualBuild"] = string(raw)
+		rd = append(rd, slsa1.ResourceDescriptor{Name: "git+" + buildDefRepo.Repo, Digest: gitDigestSet(buildDefRepo)})
```

Comment:
Should this have a prefix to differentiate it from the other resolved "git+" descriptor (inst.Location.Repo)? IIUC both entries will be bundled in the "ResolvedDependencies" section, but there's no structural way to know which git repo is the source code vs the build def but maybe I'm missing something about the structure.

Reply:
Yeah so after some research, it seems like the emerging pattern is to create a "buildConfigSource" param that includes repo/ref/path fields. I bodged together a way to get the path out of the build defn set interface so we should be able to populate that construct appropriately.